### ISSUE TITLE
Screen tearing when moving mouse pointer over controls

### ISSUE THREAD

**Issue by kkoenders** [mike-ward/tweetz#9]
When moving the mouse pointer over controls, they disform; hyperlinks become unreadable (see screenshot).

Perhaps my graphics drivers are the issue? I've got an AMD RX5700XT with the latest drivers (Radeon 20.2.2).

![afbeelding](https://user-images.githubusercontent.com/13047760/76745869-dc6aa580-6776-11ea-9b08-edef6e6a04b8.png)

**Comment by mike-ward** [comment]
That is odd. I'm using Microsoft's new .NET Core 3.1 framework, which has an updated WPF implementation. Could be driver's have not caught up yet.

**Comment by kkoenders** [comment]
I have reinstalled version 2.1.0 and with that version  I do not have this issue. I've installed v2.1.1 and the issue reappeared. Apparently something has changed between version 2.1.0 and 2.1.1 in this regard.

**Comment by mike-ward** [comment]
That does help.

There was only one impactful change in 2.1.1 - [57557e0](https://github.com/mike-ward/tweetz/commit/57557e0fb2224a2290f072c779ef0f639a4f797f)
```
<WindowChrome.WindowChrome>
    <WindowChrome
      CaptionHeight="0"
      ResizeBorderThickness="5" />
</WindowChrome.WindowChrome>
```
This restored the sizing borders instead of only sizing by using the gripper control in the lower right corner. At this point it appears to be a bug in WPF or maybe .NET Core.

**Comment by kkoenders** [comment]
The issue is solved in v2.2.1. I'll close this issue for now. Thank you for solving it :)

**Comment by mike-ward** [comment]
Ah, through the magic of doing nothing it resolves. Yeah, I rule! 😄
